src/cheatsheet.py

```diff
@@ -91,7 +93,18 @@ def run(args):
                         u'Ctrl ⏎',
                         icon=ICON_INFO)
                     action = ""
-                addShortcuts(app, action)
+                if addShortcuts(app, action) == 0:
+                    wf.add_item(nothing_found_error_text, icon=ICON_WARNING)
+    elif (u'--search-global' in args):
```

Comment:
seems to be working well. My main issue with this right now is that it's not accounting for hidden/only_shown apps. it would create an asymmetrical experience.  and global search seems like where it's most needed. 
though it's technically global, so it might make sense to keep it inclusive of all apps, or else make it configurable.
in settings.json, have: 
`include_all_app_global_search: True`
excluding hidden apps being the default behavior.
For now, I am going to merge it and create an issue for this.